be/src/Project/Services/Content/Content.Api/src/components/blocks/notification-bar.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
**suggestion (non-blocking):** Add limitation for elements in items. For example: 3